resources.ps1

```diff
@@ -0,0 +1,350 @@
+            } | ConvertTo-Json
+            Headers = @{
+                Accept         = 'application/json'
+                'Content-Type' = 'application/json'
+            }
+        }
+        
+        $tokenResponse = Invoke-RestMethod @splatTokenParams -Verbose:$false
+        Write-Output $tokenResponse.Token
+    }
+    catch {
+        $PSCmdlet.ThrowTerminatingError($_)
+    }
+}
+
+function Get-InceptionPositions {
+    [CmdletBinding()]
+    Param(
+        $pageSize = 200,
+
+        $headers
+    )
+    try {
+        $pageNumber = 1
+        $positions = [System.Collections.Generic.list[object]]::new()
+        do {
+            $splatUserParams = @{
+                Uri     = "$($actionContext.Configuration.BaseUrl)/api/v2/hrm/positions?pagesize=$pageSize&page=$pageNumber"
+                Method  = 'GET'
+                Headers = $headers
+            }
+            Write-Verbose "$($splatUserParams.Uri)"
+            $positionsResponse = Invoke-RestMethod @splatUserParams -Verbose:$false
+
+            if ($null -ne $positionsResponse.items) {
+                $positions.AddRange($positionsResponse.items)
+            }
+            $pageNumber++
+
+        }until ( $positions.count -eq $positionsResponse.total )
+        Write-Output $positions
+
+    }
+    catch {
+        $PSCmdlet.ThrowTerminatingError($_)
+    }
+}
+
+function Get-InceptionOrgunits {
+    [CmdletBinding()]
+    Param(
+        $pageSize = 200,
+        $headers
+    )
+    try {
+        $pageNumber = 1
+        $orgunits = [System.Collections.Generic.list[object]]::new()
+        do {
+            $splatUserParams = @{
+                Uri     = "$($actionContext.Configuration.BaseUrl)/api/v2/hrm/orgunits?pagesize=$pageSize&page=$pageNumber"
+                Method  = 'GET'
+                Headers = $headers
+            }
+            Write-Verbose "$($splatUserParams.Uri)"
+            $orgunitsResponse = Invoke-RestMethod @splatUserParams -Verbose:$false
+
+            if ($null -ne $orgunitsResponse.items) {
+                $orgunits.AddRange($orgunitsResponse.items)
+            }
+            $pageNumber++
+
+        }until ( $orgunits.count -eq $orgunitsResponse.total )
+        Write-Output $orgunits
+
+    }
+    catch {
+        $PSCmdlet.ThrowTerminatingError($_)
+    }
+}
+#endregion
+
+try {
+    Write-Verbose "Creating [$($resourceContext.SourceData.Count)] resources"
+    $outputContext.Success = $true
+    try {
+        <# Resource creation preview uses a timeout of 30 seconds while actual run has timeout of 10 minutes #>
+        $headers = @{
+            Accept        = 'application/json'
+            Authorization = "Bearer $(Get-InceptionToken)"
+        }
+
+        $OrgUnits = Get-InceptionOrgunits -Headers $headers
+        $positions = Get-InceptionPositions -Headers $headers
+
+
+        foreach ($resource in $resourceContext.SourceData) {
+            if ($True) {
```

Comment:
This if statement is unnecessary?

Reply:
Don't know why this is in there. Removed it cause it doesn't seem to have any functionality.